config/server.lua

```diff
@@ -1,5 +1,6 @@
 return {
-    updateInterval = 5, -- how often to update player data in minutes
+    updateInterval = 5, -- how often to update player thirst and hunger in minutes
+    dbUpdateInterval = 15, -- how often to update the database for player data in seconds
```

Comment:
The variable names should be changed to become more descriptive instead of having the comments here.

Stuff like, ```attributeUpdateIntervalMinutes``` and ```dbUpdateIntervalMinutes``` are both descriptive and completely reduce the need to comment.

Commenting should always encapsulate why instead of how.

Reply:
Considering there are no other variables that could represent something like this, a shorter name is good to make the code more readable. I don't like long names for variables.